api/accounts/accounts-router.js

```diff
@@ -1,27 +1,62 @@
+const Accounts = require(`./accounts-model`)
 const router = require('express').Router()
+const { checkAccountId, checkAccountNameUnique, checkAccountPayload } = require(`./accounts-middleware`)
 
-router.get('/', (req, res, next) => {
-  // DO YOUR MAGIC
+router.get('/', async (req, res, next) => {
+  try {
```

Comment:
Try and catch are good to use but you could also do it like: 

`
 Accounts.getAll().then(accounts => {
      res.status(200).json(accounts)
 }).catch(err => {
    next(err)
})
`